app/Navbar.tsx

```diff
@@ -26,7 +33,16 @@ export default function Navbar({ children }: { children: React.ReactNode }) {
         </div>
       </header>
 
-      <div className="text-gray-100 h-full w-screen">{children}</div>
+      <div className="w-screen h-full text-gray-100">{children}</div>
+      <div
+        className={`${
+          state.isVideoModalOpen
+            ? "h-[350px] border-t border-violet-800"
+            : "h-0"
+        } w-full bottom-0 fixed bg-stone-900  duration-300`}
+      >
+        <YoutubeVideo />
+      </div>
```

Comment:
@Birdbonez did you mean to put the YouTubeVideo in the NavBar? I think this is supposed to go in the layout.

Reply:
okay I think you put it here to 'use client' and keep layout server side.

I removed the drawer context which I think are now deprecated.